clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -458,26 +478,32 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
 
   private Closure updateRegionBackendServices(Compute compute, String project, String region, String backendServiceName, BackendService backendService) {
     return {
-      BackendService serviceToUpdate = compute.regionBackendServices().get(project, region, backendServiceName).execute()
+      BackendService serviceToUpdate = GoogleExecutor.timeExecute(
+          compute.regionBackendServices().get(project, region, backendServiceName),
+          "compute.regionBackendServices.get", TAG_SCOPE, SCOPE_REGIONAL, TAG_REGION, region)
```

Comment:
I originally wanted to do that, but it didnt look like the Java API provided the ability to do that (at least I did not see a way browsing the Javadoc). I'd prefer to do it that way. Maybe even break out the resource from the method so it is easier to filter dashboards to only look at particular resources (e.g. backendservice calls but not image calls)